netdata-installer.sh

```diff
@@ -276,13 +271,14 @@ while [ -n "${1}" ]; do
   case "${1}" in
     "--zlib-is-really-here") LIBS_ARE_HERE=1 ;;
     "--libs-are-really-here") LIBS_ARE_HERE=1 ;;
-    "--use-system-lws") USE_SYSTEM_LWS=1 ;;
-    "--dont-scrub-cflags-even-though-it-may-break-things") DONT_SCRUB_CFLAGS_EVEN_THOUGH_IT_MAY_BREAK_THINGS=1 ;;
+    "--use-system-lws") USE_SYSTEM_LWS=1; REINSTALL_OPTIONS="${REINSTALL_OPTIONS} ${1}";;
```

Comment:
Readability is difficult when we join multiple bash statements in 1 line. Can we add newlines + indentation and move the common variable (`REINSTALL_OPTIONS`) to be the first statement in this case statement? Something like:

```
"--use-system-lws")
    REINSTALL_OPTIONS="${REINSTALL_OPTIONS} ${1}"
    USE_SYSTEM_LWS=1
    ;;
```

Reply:
This change actually should not even be here...

Looks like something went wrong when I rebased.

Reply:
Wait, no, I’m wrong, this is a leftover from an early revision that I can just remove.